### DISCUSSION TITLE
Separate permissions for Series and Movies

### DISCUSSION THREAD

**Discussion by dalgardnerd** [sct/overseerr#472] [Feature Requests]
I love how great Overseerr is looking!

I would love the option to autoapprove all movie requests without auto approving a users series requests because that could easily create a huge mess if  they request old series with bad availability. I would like to curate the series that get approved but I am not so worried about movies.

Keep up the amazing work... seriously this is awesome!

**Comment by sct** [comment]
There is already a request for this here: https://github.com/sct/overseerr/issues/268

**Reply by dalgardnerd** [reply]
Woopsie. Thank you for the info
